contracts/modules/capital/MCR.sol

```diff
@@ -122,26 +120,22 @@ contract MCR is IMCR, MasterAwareV2 {
   }
 
   /*
-  * @dev trigger an MCR update. Current virtual MCR value is synced to storage, mcrFloor is potentially updated
+  * @dev trigger an MCR update. Current virtual MCR value is synced to storage
```

Comment:
I couldn't comment above, as there are no changes, but can we please rename `getAllSumAssurance()` function to `getTotalCoverAmountInEth()` or similar?